### DISCUSSION TITLE
Where is your email ? :)

### DISCUSSION THREAD

**Discussion by pierreozoux** [tiredofit/docker-collabora-online#17] [General]
Hey! We use your image since some months now, and working really great!

As a starter, feel free to request 200$ in this https://opencollective.com/libresh#category-BUDGET
(We own it, and it is in $ and I guess you prefer $, so it is easier this way.)

Then, we'd like to discuss about possible future collaboration around https://k8s.libre.sh and sponsoring.

Put your email on your github profile or update your site ;) Cheers!

**Comment by tiredofit** [comment]
Hi Pierre, very nice. My email is on my "[Front Page](https://github.com/tiredofit)", just hidden away from the spam bots. 
I do appreciate the offer and I am interested in working with you as well sharing knowledge and being a part of a team. 
I also have some up and comers that work in my organization where I act as CIO who are staunch OS/Libre supporters and contributors that I have mentored who may be of interest and value to projects..
